snapcraft/internal/meta/command.py

```diff
@@ -25,7 +26,29 @@
 from ._utils import _executable_is_valid
 from snapcraft.internal import common
 
+
+logger = logging.getLogger(__name__)
 _COMMAND_PATTERN = re.compile("^[A-Za-z0-9. _#:$-][A-Za-z0-9/. _#:$-]*$")
+_FMT_COMMAND_FROM_BASE = (
+    "The command {!r} was not found in the prime directory, it has been "
+    "changed to {!r}."
+)
+_FMT_SNAP_STRIP = (
+    "Stripped '$SNAP/' from command {!r}. "
+    "Relative paths from the prime are required by snapd."
```

Comment:
Is it really that it needs to be relative?  I figured it was just be simplified, or $VARIABLES aren't resolved?

Reply:
Yes it does, they NEED to be relative, variables are resolved, but the command's first argument needs to be **the** path itself